src/rocm_docs/core.py

```diff
@@ -75,7 +75,13 @@ class _DefaultSettings:
     copyright = _ConfigDefault("2022-2023, Advanced Micro Devices Ltd")
     # pylint: enable=redefined-builtin
     myst_enable_extensions = _ConfigUnion(
-        {"colon_fence", "fieldlist", "replacements", "substitution"}
+        {
+            "colon_fence",
+            "dollarmath",
+            "fieldlist",
+            "replacements",
+            "substitution"
```

Comment:
```suggestion
            "substitution",
            "html_image"
```
We also need HTML images, is this acceptable @samjwu